test_runner/fixtures/neon_fixtures.py

```diff
@@ -3988,8 +4030,34 @@ def clear_shared_buffers(self, cursor: Optional[Any] = None):
         """
         if cursor is not None:
```

Comment:
Need to adjust the comment above: "Might also clear LFC." part.
And probably the method itself, maybe something like `clear_shared_buffers_and_lfc`?